docs/core/tools/dotnet-add-package.md

```diff
@@ -19,22 +20,19 @@ ms.assetid: 88e0da69-a5ea-46cc-8b46-5493242b7af9
 
 ## Synopsis
 
-```
-dotnet add [project] package <package_name> [-v|--version] [-f|--framework] [-n|--no-restore] [-s|--source] [--package-directory]
-dotnet add package [-h|--help]
-```
+`dotnet add [<PROJECT>] package <PACKAGE_NAME> [-v|--version <VERSION>] [-f|--framework <FRAMEWORK>] [-n|--no-restore] [-s|--source <SOURCE>] [--package-directory <PACKAGE_DIRECTORY>] [-h|--help]`
```

Comment:
I was following an existing pattern where if it was optional it has brackets; but if optional and user-supplied, it would get both brackets and angle brackets.

If you prefer it be brackets only, I'll update it everywhere. A bunch of the commands have this.